diff -up paperwork-2.1.2/setup.py.orig paperwork-2.1.2/setup.py
--- a/paperwork-2.1.2/setup.py	2023-07-22 22:41:37.787416023 -0400
+++ b/paperwork-2.1.2/setup.py	2023-07-22 22:42:07.987688285 -0400
@@ -90,7 +90,7 @@ Main features are:
         "openpaperwork-gtk",
         "pycountry",
         "pyocr >= 0.3.0",
-        "python-Levenshtein",
+        "Levenshtein",
         "pyxdg >= 0.25",
         "paperwork-backend>={}".format(version),
 
diff -up paperwork-2.1.2/src/paperwork.egg-info/requires.txt.orig paperwork-2.1.2/src/paperwork.egg-info/requires.txt
--- a/paperwork-2.1.2/src/paperwork.egg-info/requires.txt	2023-07-23 14:56:07.332088123 -0400
+++ b/paperwork-2.1.2/src/paperwork.egg-info/requires.txt	2023-07-23 14:56:13.424146746 -0400
@@ -4,5 +4,5 @@ openpaperwork-gtk
 paperwork-backend>=2.1.2
 pycountry
 pyocr>=0.3.0
-python-Levenshtein
+Levenshtein
 pyxdg>=0.25
